docs/application/native/guides/app-management/ui-app/efl-app.md

```diff
@@ -10,19 +10,24 @@ EFL is a collection of libraries that are independent or can build on top of eac
 Before you start UI programming with EFL, you must familiarize yourself with the basic EFL fundamentals. EFL consists of the following main libraries:
```

Comment:
On line 4, the link, `[Dynamic Animation Library (DALi) UI toolkit](../ui/dali/dali-overview.md)` is broken. It looks like `(../../ui/dali/index.md)` is correct.

Reply:
done